.golangci.yml

```diff
@@ -23,7 +23,6 @@ linters:
     - importas
     - ineffassign
     - misspell
-    - nakedret
```

Comment:
I have no preference on naked vs not-naked returns, but could not see a strong enough need in this pr to change the linter configuration, and consequently the coding policy. The changes with naked returns have no functional requirement, while naked returns could be a source of a subtle problems.